build.py

```diff
@@ -50,22 +51,25 @@
 	'-Wno-int-conversion',
 	'-Wno-unused-function',
 	'-I${SYSROOT}/usr/include/libxml2',
+        '-I/usr/include/curl',
 	'-I./',
 ])
 
 swift_cflags = [
 	'-I${BUILD_DIR}/Foundation/usr/lib/swift',
-	'-I${SYSROOT}/usr/include/libxml2'
+	'-I${SYSROOT}/usr/include/libxml2',
+        '-I/usr/include/curl'
 ]
 
 if "XCTEST_BUILD_DIR" in Configuration.current.variables:
 	swift_cflags += [
 		'-I${XCTEST_BUILD_DIR}',
 		'-L${XCTEST_BUILD_DIR}',
-		'-I${SYSROOT}/usr/include/libxml2'
+		'-I${SYSROOT}/usr/include/libxml2',
+                '-I/usr/include/curl'
```

Comment:
This should also be `${SYSROOT}` attached just like the libxml case so that way cross compile development can specify a root